alibi_detect/cd/classifier.py

```diff
@@ -122,8 +123,9 @@ def __init__(
             Optional additional kwargs when fitting the classifier. Only relevant for 'tensorflow' and
             'pytorch' backends.
         device
-            Device type used. The default None tries to use the GPU and falls back on CPU if needed.
-            Can be specified by passing either 'cuda', 'gpu' or 'cpu'. Only relevant for 'pytorch' backend.
+            Device type used. The default tries to use the GPU and falls back on CPU if needed.
+            Can be specified by passing either ``'cuda'``, ``'gpu'``, ``'cpu'`` or an instance of
+            ``torch.device``. Only relevant for 'pytorch' backend.
```

Comment:
Just out of curiosity, if you update the `intersphinx_mapping` like https://github.com/pytorch/pytorch/issues/10400 and then reference `torch.device` like `:py:class:torch.device`  (should be in backticks), does it work?

Reply:
Can I leave this to explore in a separate issue? This PR already has a much wider scope than initially intended! :sweat_smile: